docs/_config.yml

```diff
@@ -9,7 +9,9 @@ ALLUXIO_RELEASED_VERSION: 1.4.0-RC1-PS
 # For example, "master" for the master branch, and 1.1 for 1.1.x releases and release candidates.
 ALLUXIO_MASTER_VERSION_SHORT: 1.4.0-RC1-PS
 
-ALLUXIO_CLIENT_JAR_PATH: /<PATH_TO_ALLUXIO>/core/client/target/alluxio-core-client-{{site.ALLUXIO_RELEASED_VERSION}}-jar-with-dependencies.jar
+# We must inline the version string (e.g., "1.4.0-SNAPSHOT") rather than using the macro defined above.
+# Otherwise the macro name remains in the output.
+ALLUXIO_CLIENT_JAR_PATH: /<PATH_TO_ALLUXIO>/core/client/target/alluxio-core-client-1.4.0-RC1-jar-with-dependencies.jar
```

Comment:
BTW, is this change intended? if so, the inlined version does not matched the above RELEASED_VERSION